CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+
+# Contributing to Razor Network Datasources
+If you wish to propose a new price feed to be added to the datasources used by the protocol, please follow the steps below:
+
+1. Start by forking the repository and creating your branch from `master`.
+2. Add the new job to `mainnet/jobs.json` and update `mainnet/collections.json` as needed with the correct jobIDs. (an example for `testnet` is given below)
+    - For example, adding a new `ETH/USD` job to an existing collection for testnet:
+```json
+[
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "ethusd_gemini",
+    "selector": "last",
+    "url": "https://api.gemini.com/v1/pubticker/ethusd"
+  },
+  ... ( 8 other entries here) ...,
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "ethusd_test",
+    "selector": "selector[0]",
+    "url": "https://www.example.com/"
+  }
+]
+```
+Now to include this new job into an existing collection in `testnet/collections.json` we need to add the serialized jobID to the desired collection. In this case we are adding `ETH/USD` job to the existing `ethCollectionMedian` collection:
+
+```json
+[
+  {
+    "jobIDs": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], // 10 is added here which is the jobId of the new  ethusd_test job
```

Comment:
Please explain the attributes

Reply:
noted

Reply:
resolved here [6b4dd2c](https://github.com/razor-network/datasources/pull/55/commits/6b4dd2c39021acbfdeaa19e752379cd5d63f9345)